### PR TITLE
Changes in the spec file so rhel7 build works

### PR DESCRIPTION
**Pull request by efajardo**

### CODE REVIEW

File: spec/xrootd-lcmaps.spec [hunk 1]
```diff
@@ -21,11 +22,16 @@ Requires: xrootd-server >= 1:3.2
 %{summary}
 
 %prep
-%setup -q -c -n %{name}-%{version}
+#%setup -q -c -n %{name}-%{version}
+%setup -q -c -n %{name}
```
Comment (bbockelm) [inline spec/xrootd-lcmaps.spec]: No - I think this is right the first time.  The tarball should unpack into xrootd-lcmaps-$VERSION.  Can you undo this change?

File: spec/xrootd-lcmaps.spec [hunk 2]
```diff
@@ -21,11 +22,16 @@ Requires: xrootd-server >= 1:3.2
...
 
 %build
 #cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=RelWithDebInfo .
+%if 0%{?rhel} > 6
+%cmake -DCMAKE_INSTALL_LIBDIR:PATH=/usr/lib64 -DCMAKE_BUILD_TYPE=RelWithDebInfo .
```
Comment (bbockelm) [inline spec/xrootd-lcmaps.spec]: Why was this necessary?

Comment (efajardo) [inline spec/xrootd-lcmaps.spec]: The build for rhel7 was failing with this error:

RPM build errors:
    File not found: /builddir/build/BUILDROOT/xrootd-lcmaps-1.0.0-1.osg33.el7.x86_64/usr/lib64/libXrdLcmaps.so
    File not found: /builddir/build/BUILDROOT/xrootd-lcmaps-1.0.0-1.osg33.el7.x86_64/usr/lib64/libXrdLcmaps.so.0
    File not found: /builddir/build/BUILDROOT/xrootd-lcmaps-1.0.0-1.osg33.el7.x86_64/usr/lib64/libXrdLcmaps.so.0.0.2

And it was because of the CMAKE args. This is for el7:

 /usr/bin/cmake -DCMAKE_C_FLAGS_RELEASE:STRING=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE:STRING=-DNDEBUG -DCMAKE_Fortran_FLAGS_RELEASE:STRING=-DNDEBUG -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DCMAKE_INSTALL_PREFIX:PATH=/usr -DINCLUDE_INSTALL_DIR:PATH=/usr/include -DLIB_INSTALL_DIR:PATH=/usr/lib64 -DSYSCONF_INSTALL_DIR:PATH=/etc -DSHARE_INSTALL_PREFIX:PATH=/usr/share -DLIB_SUFFIX=64 -DBUILD_SHARED_LIBS:BOOL=ON -


And this one for el6:

/usr/bin/cmake -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_PREFIX:PATH=/usr -DCMAKE_INSTALL_LIBDIR:PATH=/usr/lib64 -DINCLUDE_INSTALL_DIR:PATH=/usr/include -DLIB_INSTALL_DIR:PATH=/usr/lib64 -DSYSCONF_INSTALL_DIR:PATH=/etc -DSHARE_INSTALL_PREFIX:PATH=/usr/share -DLIB_SUFFIX=64 -DCMAKE_SKIP_RPATH:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo .

SI had to add the missing:

-DCMAKE_INSTALL_LIBDIR:PATH=/usr/lib64 for el7

File: spec/xrootd-lcmaps.spec (hunk 1 above)
Comment (efajardo) [inline spec/xrootd-lcmaps.spec]: Not really:

To get the upstream I used the following:

```bash
git clone https://github.com/bbockelm/xrootd-lcmaps.git
cd xrootd-lcmaps
git-archive v1.0.0 | gzip -7 > ~/xrootd-lcmaps.tar.gz
```

And then it unpacks into xrootd-lcmaps

Comment (bbockelm) [inline spec/xrootd-lcmaps.spec]: Ah, try changing that to:

```
git archive v1.0.0 --prefix=xrootd-lcmaps-v1.0.0 | gzip -7 > ~/xrootd-lcmaps-1.0.0.tar.gz
```

File: spec/xrootd-lcmaps.spec (hunk 2 above)
Comment (bbockelm) [inline spec/xrootd-lcmaps.spec]: Can you instead switch the `CMakeLists.txt` to use `LIB_INSTALL_DIR`?  That appears to work on both.
